assets/src/blocks/SecondaryNavigation/SecondaryNavigationBlock.js

```diff
@@ -0,0 +1,39 @@
+import {SecondaryNavigationEditor} from './SecondaryNavigationEditor';
+import {example} from './example';
+
+const {__} = wp.i18n;
+
+const BLOCK_NAME = 'planet4-blocks/secondary-navigation';
+
+export const registerSecondaryNavigationBlock = () => {
+  const {registerBlockType} = wp.blocks;
+
+  registerBlockType(BLOCK_NAME, {
+    title: 'Secondary Navigation Menu',
+    description: __('Inserts a secondary navigation menu to the page that leads to different sections of the same page.', 'planet4-blocks-backend'),
+    icon: 'menu-alt3',
+    category: 'planet4-blocks-beta',
+    attributes: {
+      levels: {
+        type: 'array',
+        default: [{heading: 2, link: true}],
```

Comment:
Is there an option to change this?

Reply:
I will assume at some point we may want to use this for different heading levels, maybe give editors the option to use it for h2 only or h3 only